airbyte-workers/src/main/java/io/airbyte/workers/RecordSchemaValidationException.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers;
+
+public class RecordSchemaValidationException extends Exception {
```

Comment:
Could you add a brief javadoc to this class explaining what it means / what can cause this?